phoenix-core/src/main/java/org/apache/phoenix/coprocessor/tasks/IndexRebuildTask.java

```diff
@@ -107,6 +107,7 @@
             Task.addTask(conn.unwrap(PhoenixConnection.class ), taskRecord.getTaskType(), taskRecord.getTenantId(), taskRecord.getSchemaName(),
                     taskRecord.getTableName(), PTable.TaskStatus.STARTED.toString(), jsonObject.toString(), taskRecord.getPriority(),
                     taskRecord.getTimeStamp(), null, true);
+            conn.commit();
```

Comment:
@gokceni what are we committing here? Also, this change seems outside the scope of this JIRA again. Is there any change you made which would now require a `conn.commit` after adding a task?

Reply:
removed